templates/hpc/contact-us.html

```diff
@@ -0,0 +1,117 @@
+
+  {% elif product == 'hpc-maas' %}
+
+{% with h1="Contact us about MAAS for cluster management",
+  intro_text="Fill in your details below and a member of our data center sales team will be in touch.	",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=hpc-maas" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+  {% elif product == 'hpc-kubernetes' %}
+
+{% with h1="Contact us about Kubernetes for workload management",
+  intro_text="Fill in your details below and a member of our data center sales team will be in touch.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=hpc-kubernetes" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+    {% elif product == 'hpc-ceph' %}
+
+{% with h1="Contact us about Ceph for Computational Storage",
+  intro_text="Fill in your details below and a member of our data center sales team will be in touch.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=hpc-ceph" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+  {% elif product == 'hpc-openstack' %}
+
+{% with h1="Contact us about OpenStack & HPC",
+  intro_text="Fill in your details below and a member of our data center sales team will be in touch.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=hpc-openstack" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+{% elif product == 'hpc-isv' %}
+
+{% with h1="As a HPC & Supercomputing ISV let’s discuss partnership",
+  intro_text="Fill in your details below and a member of our alliance team will be in touch.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=hpc-isv" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+{% elif product == 'openinfra-summit' %}
+
+{% with h1="Let’s talk at OpenInfra Summit",
+  intro_text="Fill in your details below and a member of our team will be in touch to arrange a meeting.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=openinfra-summit" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+{% elif product == 'supercomputing' %}
+
+{% with h1="Let’s talk at Supercomputing",
+  intro_text="Fill in your details below and a member of our team will be in touch to arrange a meeting.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=supercomputing" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+
+{% elif product == 'isc-hpc' %}
+
+{% with h1="Let’s talk at ISC HPC",
+  intro_text="Fill in your details below and a member of our team will be in touch to arrange a meeting.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=isc-hpc" %}
+  {% include "hpc/_hpc-contact-us-form.html" %}
+{% endwith %}
+
+  {% else %}
+
+{% with h1="Contact us about HPC & Supercomputing",
+  intro_text="Fill in your details below and a member of our data center sales team will be in touch.",
+  formid="4537",
+  returnURL="/openstack/thank-you?product=hpc-demo" %}
```

Comment:
```suggestion
  returnURL="/openstack/thank-you" %}
```

Reply:
Oh wow, I didn't notice this in the QA, actually this should be `/hpc/thank-you`. Thanks